.vscode/settings.json

```diff
@@ -4,5 +4,9 @@
     "gopls": {
         "formatting.local": "github.com/G-Research/fasttrackml",
         "formatting.gofumpt": true
-    }
+    },
+    "go.lintTool": "golangci-lint",
+    "go.lintFlags": [
+        "--fast"
```

Comment:
maybe just add `-v` for more information?

Reply:
also -> ` --tests Analyze tests (*_test.go) (default true)` will be nice to disable `test` linting, because sometimes it could be a pain to follow all the rules, buuut. what do you think?

Reply:
also, thinking about next linters to enable:
- [prealloc](https://github.com/alexkohler/prealloc)
- [varnamelen](https://github.com/blizzy78/varnamelen)
- [nilerr](https://github.com/gostaticanalysis/nilerr)
- [nestif](https://github.com/nakabonne/nestif)
- [misspell](https://github.com/client9/misspell)
- [ireturn](https://github.com/butuzov/ireturn)
- [gomnd](https://github.com/tommy-muehle/go-mnd)
- [godot](https://github.com/tetafro/godot)

Reply:
`-v` does not seem to make a difference in VS Code. In both cases, linting errors look like this:
![image](https://github.com/G-Research/fasttrackml/assets/3692455/83d74ba1-dc93-4aa2-84ad-cc65043370a7)

Reply:
about `--tests`, I think let's keep it for now — ideally we do want to follow all the rules, but we'll see once we actually enforce it how it goes!